waku/node/v2/rpc/rpc_info.nim

```diff
@@ -0,0 +1,19 @@
+import
+  os, strutils, strformat, chronicles, json_rpc/[rpcclient, rpcserver], nimcrypto/sysrand,
+  libp2p/protobuf/minprotobuf,
+  libp2p/[peerinfo, multiaddress],
+  eth/common as eth_common, eth/keys,
+  system,
+  options
+
+from strutils import rsplit
+template sourceDir: string = currentSourcePath.rsplit(DirSep, 1)[0]
+
+const sigWakuPath = sourceDir / "wakucallsigs.nim"
+createRpcSigs(RpcHttpClient, sigWakuPath)
+
+var node = newRpcHttpClient()
+waitfor node.connect("localhost", Port(8545))
+
+var res = waitfor node.wakuInfo()
```

Comment:
Would be better to change this into a test.

Reply:
Yeah agree, was on my TODO but decided to skip it for now (a) to get it merged (b) until we know what we want with JSON RPC (c) until there is more meat in the WakuInfo call